### ISSUE TITLE
Thers is no core??

### ISSUE THREAD

**Issue by leisongju** [KAIST-VCLAB/dual-pixel-disparity#1]
from core.update import BasicMultiUpdateBlock
from core.extractor import BasicEncoder, MultiBasicEncoder, ResidualBlock
from core.corr import CorrBlock1D
from core.utils.utils import upflow8
from core.raft_stereo import RAFTStereo

where is the core???

**Comment by michael-1003** [comment]
If you follow the install instructions in section 1, it will automatically download the source from externals (RAFT-stereo) and there is "core/...". Otherwise, if you need a manual download, see the download script in `env/install.sh`.

**Comment by leisongju** [comment]
thank you，Is there any training code here? Mainly the loss function part

**Comment by michael-1003** [comment]
No. Right now, the training code is not here.

**Comment by leisongju** [comment]
> No. Right now, the training code is not here.

Will you make your training code public?
